apps-rendering/package-lock.json

```diff
@@ -2789,9 +2789,9 @@
       }
     },
     "@guardian/atoms-rendering": {
-      "version": "22.0.0",
-      "resolved": "https://registry.npmjs.org/@guardian/atoms-rendering/-/atoms-rendering-22.0.0.tgz",
-      "integrity": "sha512-fH0E8cTn8XUo1Mc0ft6M2S0vjuREcVayypkVzfBZ74N7hbkgq80ZFWnOJ6Vi730XI5y7NUXnvRz/7XhKV2YSIQ==",
+      "version": "22.6.3",
+      "resolved": "https://registry.npmjs.org/@guardian/atoms-rendering/-/atoms-rendering-22.6.3.tgz",
+      "integrity": "sha512-pHsYi+drZ5LU6AWlV1FSrOh5i1DUV5iMAD8m6kJYkl6U3uUYFV2J9bBkZrJYTniuONzARGcKvbxtqGZiX0db4g==",
```

Comment:
This change makes sense to me. Just to confirm, are the rest of the changes in this file also coming from the new `atoms-rendering` version?